main.go

```diff
@@ -94,4 +94,13 @@ func main() {
 		parameters.ConfigureParameters(&context)
 		os.Exit(0)
 	}
+
+	if *context.CliArguments.Mode == cliparser.SetStackPolicyMode {
+		if *context.CliArguments.DisableStackTermination || *context.CliArguments.EnableStackTermination {
+			stack.SetTerminationProtection(&context)
+		} else {
+			stack.ApplyStackPolicy(&context)
+		}
+		os.Exit(0)
```

Comment:
Please handle `SetTerminationProtection` and `ApplyStackPolicy` errors here and return exit code other than 0 if any error happen.